lib/ffmpeg/movie.rb

```diff
@@ -52,8 +52,8 @@ def initialize(path)
       end
 
       if audio_stream
-        @audio_codec, audio_sample_rate, @audio_channels, unused, audio_bitrate = audio_stream.split(/\s?,\s?/)
-        @audio_bitrate = audio_bitrate =~ %r(\A(\d+) kb/s(.*)\Z) ? $1.to_i : nil
+        @audio_codec, audio_sample_rate, @audio_channels, _unused, audio_bitrate = audio_stream.split(/\s?,\s?/)
+        @audio_bitrate = audio_bitrate =~ %r((\d+) kb/s) ? $1.to_i : nil
```

Comment:
This is the change related to how more recent versions of ffmpeg (ffprobe) deliver the audio bitrate. It may not work for older versions.